CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ SPDX-License-Identifier: GPL-3.0-or-later
 -->
 
 # 0.5.4 (unreleased)
+- Remove unnecessary semicolon in source code [PR #131] (https://github.com/hpsim/OGL/pull/131)
```

Comment:
Not needed to add to the changelog since it has no effect for the user.
```suggestion
```

Reply:
No worries, we can still merge. It is not a hard requirement. We could also add an optional flag for the changelog test.